python_modules/dagster-ext/dagster_ext/__init__.py

```diff
@@ -94,6 +96,23 @@ class ExtDataProvenance(TypedDict):
     is_user_provided: bool
 
 
+ExtMetadataType = Literal[
+    "text",
+    "url",
+    "path",
+    "notebook",
+    "json",
+    "md",
+    "float",
+    "int",
+    "bool",
+    "dagster_run",
+    "asset",
+    "table",
+    "null",
+]
```

Comment:
where do we guarantee that these all line up to valid string literals in dagster core?

Reply:
they are now tested